lib/models/builder.js

```diff
@@ -66,6 +66,9 @@ class Builder extends CoreObject {
       };
     } else {
       broccoli = require('broccoli-builder');
+      if (this.systemTemp) {
+        console.warn('EMBER_CLI_SYSTEM_TEMP only works in combination with EMBER_CLI_BROCCOLI_2');
```

Comment:
we should simply error here

Reply:
Will need to isolate tests in that case, because we can't do only combo environment variables (e.g. BROCCOLI_2 and SYSTEM_TEMP). Right now, the SYSTEM_TEMP flag will cause this error when being run without setting BROCCOLI_2.
Will do a little refactor.

Reply:
Done